bibliopixel/util/importer.py

```diff
@@ -1,21 +0,0 @@
-import importlib
```

Comment:
LOL... Damnit, BPT imports import_symbol to do some tests. Oops. Will have to change in BPT. What's the replacement for this?